src/mono/wasm/runtime/startup.ts

```diff
@@ -386,10 +394,6 @@ async function mono_wasm_after_user_runtime_initialized(): Promise<void> {
                 }
             }
         }
-        // for Blazor, init diagnostics after their "onRuntimeInitalized" sets env variables, but before their postRun callback (which calls mono_wasm_load_runtime)
-        if (MonoWasmThreads) {
-            await mono_wasm_init_diagnostics();
-        }
```

Comment:
Is this still true in the new code? Is `mono_wasm_before_user_runtime_initialized` after blazor's `onRuntimeInitialized` and before their `postRun`?